### PR TITLE
Update BaseSamplePrinter.java

### PR DESCRIPTION
**Pull request by winsonrich**
modifier is added.

### CODE REVIEW

**Comment by winsonrich** [comment]
I think the procedure of initializing the variables should not put in the else{} case which may cause run time error.

File: andhow-core/src/main/java/org/yarnandtail/andhow/sample/BaseSamplePrinter.java [hunk 1]
```diff
@@ -43,16 +43,16 @@ protected void print(PrintStream out, TextBlock block, PrintFormat format) {
 		
 		if (block == null) return;
 		
-		boolean useBlockComment = false;
-		boolean useLineComment = false;
+		private boolean useBlockComment = false;
+		private boolean useLineComment = false;
```
Comment (eeverman) [inline andhow-core/src/main/java/org/yarnandtail/andhow/sample/BaseSamplePrinter.java]: This is invalid syntax - You cannot declare scope on a local variable.  I'm guessing this is why the AppVeyor compilation is failing.

File: andhow-core/src/main/java/org/yarnandtail/andhow/sample/BaseSamplePrinter.java [hunk 2]
```diff
@@ -148,8 +148,8 @@ public void printPropertyGroupStart(StaticPropertyConfigurationInternal definiti
 		tb.setBlankLineAfter(true);
 		tb.addHR();
 		
-		String name = null;
-		String desc = null;
+		private String name = null;
+		private String desc = null;
```
Comment (eeverman) [inline andhow-core/src/main/java/org/yarnandtail/andhow/sample/BaseSamplePrinter.java]: Same - no scope on local variables.

File: andhow-core/src/main/java/org/yarnandtail/andhow/sample/BaseSamplePrinter.java [hunk 3]
```diff
@@ -189,7 +189,7 @@ public void printProperty(StaticPropertyConfigurationInternal definition, PrintS
 		
 		try {
 			
-			String propFieldName = group.getSimpleName(prop);
+			private String propFieldName = group.getSimpleName(prop);
```
Comment (eeverman) [inline andhow-core/src/main/java/org/yarnandtail/andhow/sample/BaseSamplePrinter.java]: same.

**Comment by eeverman** [comment]
Since these changes do not help the project and cause the build to fail, I am going to close this pull request.

**Pull request closed without merging**